app/mailers/due_soon_and_overdue_defects_mailer.rb

```diff
@@ -1,5 +1,11 @@
 class DueSoonAndOverdueDefectsMailer < ApplicationMailer
   def notify(defect_ids)
-    @defects = Defect.find(defect_ids)
+    @defects = Defect.find(defect_ids).map { |defect| DefectPresenter.new(defect) }
```

Comment:
I thought usually the pattern with presenters is to have a `DefectPresenters` class to do all this mapping, so you could just do `@defects = DefectsPresenter.new(Defect.find(defect_ids))`

Reply:
I used the same pattern here as is used in the `DefectsController`. We don't have a pattern of presenters for collections in the system... yet.

If you're okay with it, I'd like to take a note to change this, but not do it as part of this PR. Since this new `DefectsPresenter` would touch more than these emails.

Reply:
Yeah, that's cool 👍